content/en/observability_pipelines/sources/azure_event_hubs.md

```diff
@@ -0,0 +1,110 @@
+
+### Configure Kafka-compatible connection for the event hub
+
+Azure Event Hubs exposes a Kafka endpoint at `NAMESPACE.servicebus.windows.net:9093`, which Observability Pipelines uses as the Kafka source.
+
+#### Get the Kafka endpoint
+
+1. In the Azure Portal, navigate to your Event Hubs Namespace (for example, `myeventhubns`).
+1. On the **Overview** page, under the **Essentials** section, locate the **Host name** or **Fully Qualified Domain Name (FQDN)**. It is in the format: `<NAMESPACE>.servicebus.windows.net` (for example, `myeventhubns.servicebus.windows.net`).
+1. Append the Kafka port `:9093` to form the Bootstrap Servers value: `<NAMESPACE>.servicebus.windows.net:9093`.
+    - For example, if your namespace is `myeventhubns`, the Bootstrap Servers is `myeventhubns.servicebus.windows.net:9093`.
+    - You need this information when you set up the Observability Pipelines Kafka source.
+
+#### Set up authentication
+
+1. Azure Event Hubs uses SASL_SSL with the PLAIN mechanism for Kafka authentication.
+1. The connection string is formatted for Observability Pipelines:
+    ```
+    Username: $ConnectionString
+    Password: Endpoint=sb://<NAMESPACE>.servicebus.windows.net/;SharedAccessKeyName=<PolicyName>;SharedAccessKey=<Key>
+    ```
+
+## Set up a pipeline with the Kafka source
+
+1. Navigate to [Observability Pipelines](https://app.datadoghq.com/observability-pipelines).
+1. Select the Kafka source.
+    1.  In the **Group ID** field, specify or create a unique consumer group (for example, `datadog-consumer-group`).
+    1.  Enter `datadog-topic` in the **Topics** field.
+    1.  Toggle the switch to enable SASL authentication.
+    1.  In the **Mechanism** dropdown menu, select **PLAIN**.
+    1.  Enable TLS.
+        1.  Download the certificate from [https://curl.se/docs/caextract.html](https://curl.se/docs/caextract.html) and save it to `/var/lib/observability-pipelines-worker/config/cert.pem`.
+        1.  Enter `/cert.pem` in the **Certificate path** field.
+1. Click **Next: Select Destination**.
+1. After you set up your destinations and processors, click **Next: Install**.
+1. Select your platform in the **Choose your installation platform** dropdown menu.
+1. Enter the environment variables for your Kafka source:
+    1.  For **Kafka Bootstrap Servers**, enter `\<NAMESPACE\>.servicebus.windows.net:9093` (for example, `myeventhubns.servicebus.windows.net:9093`).
+    1.  For **Kafka SASL Username**, enter `$ConnectionString`.
+    1.  For **Kafka SASL Password**, enter the full connection string (for example, `Endpoint=sb://<NAMESPACE>.servicebus.windows.net/;SharedAccessKeyName=<PolicyName>;SharedAccessKey=<Key>`).
+    1. Enter your Kafka TLS passphrase.
+1. Enter the environment variables for your destinations, if applicable.
+1. Follow the rest of the instructions on the page to install the Worker based on your platform.
+
+#### Configure the Observability Pipelines environment file
```

Comment:
This will be covered by the install steps in app when they run the install command, they should only need to do this manually if something went wrong with the install and they need to modify it after the fact, so maybe we should re-word this to say "Check your Observability Pipelines environments file"? or add some additional flavor text about editing this post install should you need to?

Reply:
I made a suggestion below, is that correct? Should they see those env vars and their values in the environment file after they have installed the Worker?

Reply:
Yes they should, LGTM